scripts/tests/provisioning/create_safemode.expect

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/expect -f
```

Comment:
Can we rename this to something more explicit since this is not really creating a safemode? It will make it easier to remember what this does in the future. Something like 'test-efi-ab-to-safemode.expect' or 'test-rauc-to-safemode.expect'. FWIW I also don't like the existing generically named  'test.expect'. I might try to rename that in the PR that will go with my changes.

Reply:
This file is intended for just creating safemode VM not testing. There is another expect file for actually running the ptest.
I was planning to run the create_safemode.expect in azdo subcomponent that builds the gateway ipk, and run the actual test expect file in another subcomponent that does the testing. This way your test subcomponent can consume the safemode VM directly.

Reply:
OK that makes sense but it's still not creating safemode. Maybe create_safemode_vm.expect then? Or more accurately provision_safemode_vm.expect or migrate_vm_to_safemode.expect.